travisciapiprovider.py

```diff
@@ -0,0 +1,16 @@
+import urllib
+import json
+
+# If more functionality is needed from this class, it might be a
+# better decision to use travispy
+class TravisCiApiProvider():
+    host_url = 'https://api.travis-ci.org'
```

Comment:
We would have a json file that contains a travis payload instead of a github payload like existing tests, and we would check for the presence of the S-needs-code-changes label in the expected output.